bundle/manifests/dell-csm-operator.clusterserviceversion.yaml

```diff
@@ -3731,17 +3731,17 @@ spec:
   relatedImages:
   - image: docker.io/dellemc/dell-csm-operator:v1.5.1
     name: dell-csm-operator
-  - image: docker.io/dellemc/csi-isilon:v2.9.1
+  - image: docker.io/dellemc/csi-isilon:v2.10.1
     name: csi-isilon
-  - image: docker.io/dellemc/csi-powermax:v2.9.1
+  - image: docker.io/dellemc/csi-powermax:v2.10.1
     name: csi-powermax
   - image: docker.io/dellemc/csipowermax-reverseproxy:v2.8.1
     name: csipowermax-reverseproxy
-  - image: docker.io/dellemc/csi-powerstore:v2.9.1
+  - image: docker.io/dellemc/csi-powerstore:v2.10.1
     name: csi-powerstore
-  - image: docker.io/dellemc/csi-unity:v2.9.1
+  - image: docker.io/dellemc/csi-unity:v2.10.1
     name: csi-unity
-  - image: docker.io/dellemc/csi-vxflexos:v2.9.1
+  - image: docker.io/dellemc/csi-vxflexos:v2.10.1
     name: csi-vxflexos
   - image: docker.io/dellemc/sdc:4.5.1
     name: sdc
```

Comment:
@chimanjain Can you please confirm if SDC is going to be same for patch release ?

Reply:
@chimanjain, @rajendraindukuri: SDC version will be same for this patch release

Reply:
Thanks @suryagupta4